server.js

```diff
@@ -76,10 +77,8 @@ app.prepare().then(() => {
   server.use("/auth", authRouter);
   server.use("/api/profile", profileRouter);
   server.use("/api/admin", adminRouter);
+  server.use("/api/team", teamRouter);
 
-  server.get("/dashboard/:step", (req, res) =>
-    app.render(req, res, "/dashboard")
-  );
```

Comment:
Do we not need this to render the dashboard?

Reply:
nah we don't, this was leftover old code back from when we rendered steps via those dashboard tabs